packages/proxmark3-git/build.sh

```diff
@@ -1,11 +1,12 @@
 TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
 TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
-TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
-_COMMIT=c41eab99c0ae3636bf095a741d95d1b0395c6db7
-TERMUX_PKG_VERSION="2022.02.10-${_COMMIT:0:8}"
-TERMUX_PKG_SRCURL="https://github.com/RfidResearchGroup/proxmark3/archive/c41eab99c0ae3636bf095a741d95d1b0395c6db7.tar.gz"
-TERMUX_PKG_SHA256="26f854fc0ee681b8a6c890902a209aa63579f3a68271b9f224ba27eea4107391"
+TERMUX_PKG_REVISION=1
```

Comment:
Please reset `TERMUX_PKG_REVISION` when bumping version.

Reply:
Not needed for license update?